deltachat-ffi/deltachat.h

```diff
@@ -4669,7 +4669,7 @@ void dc_event_unref(dc_event_t* event);
  * Inform about the configuration progress started by dc_configure().
  *
  * @param data1 (int) 0=error, 1-999=progress in permille, 1000=success and done
- * @param data2 0
```

Comment:
DC_EVENT_DATA2_IS_STRING() needs to be adapted, at least android still relies on that.

Reply:
Updated it. Does Android deal correctly with the case when the string is NULL?

Reply:
yip, i just double-checked that. however, we should document that data2 can be null.